src/app/+browse-by/+browse-by-metadata-page/browse-by-metadata-page.component.ts

```diff
@@ -104,6 +123,11 @@ export class BrowseByMetadataPageComponent implements OnInit {
           }
           this.updateParent(params.scope);
         }));
+    this.updateStartsWithTextOptions();
+  }
+
+  updateStartsWithTextOptions() {
```

Comment:
Please add comments to describe this function

Reply:
Added comments.

Reply:
👍